<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1160</width>
    <height>719</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CT上窦额分割</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="readButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>读取数据</string>
    </property>
   </widget>
   <widget class="QPushButton" name="surfaceRenderButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>580</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>面绘制</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resetButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>460</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>重置</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>470</y>
      <width>381</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="threshouldButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>460</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>阈值分割</string>
    </property>
   </widget>
   <widget class="QScrollBar" name="horizontalScrollBar">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>530</y>
      <width>160</width>
      <height>21</height>
     </rect>
    </property>
    <property name="pageStep">
     <number>5</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="volumeRenderButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>580</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>体绘制</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>570</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>三维重建</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>0</y>
      <width>370</width>
      <height>677</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QVTKWidget" name="qvtkWidget_Volume_2" native="true">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>343</width>
           <height>219</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QScrollBar" name="verticalScrollBar">
         <property name="maximum">
          <number>511</number>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QVTKWidget" name="qvtkWidget_Volume_3" native="true">
         <property name="minimumSize">
          <size>
           <width>343</width>
           <height>219</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QScrollBar" name="verticalScrollBar_2">
         <property name="maximum">
          <number>511</number>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QVTKWidget" name="qvtkWidget_Volume_4" native="true">
         <property name="minimumSize">
          <size>
           <width>343</width>
           <height>219</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QScrollBar" name="verticalScrollBar_3">
         <property name="maximum">
          <number>200</number>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="contourSegmentButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>460</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>分割上窦额</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>520</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="documentTitle">
     <string/>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;-512&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="acceptRichText">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>530</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>分割阈值</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>13</y>
      <width>708</width>
      <height>402</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QVTKWidget" name="qvtkWidget_Volume_6" native="true">
       <property name="minimumSize">
        <size>
         <width>350</width>
         <height>400</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QVTKWidget" name="qvtkWidget_Volume" native="true">
       <property name="minimumSize">
        <size>
         <width>350</width>
         <height>400</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="denoiseButton">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>430</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>去噪</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>430</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_2">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>430</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_3">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>430</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_4">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>430</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <zorder>layoutWidget</zorder>
   <zorder>readButton</zorder>
   <zorder>surfaceRenderButton</zorder>
   <zorder>resetButton</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>textBrowser</zorder>
   <zorder>threshouldButton</zorder>
   <zorder>horizontalScrollBar</zorder>
   <zorder>volumeRenderButton</zorder>
   <zorder>label</zorder>
   <zorder>contourSegmentButton</zorder>
   <zorder>textBrowser_2</zorder>
   <zorder>label_2</zorder>
   <zorder>denoiseButton</zorder>
   <zorder>textEdit</zorder>
   <zorder>textEdit_2</zorder>
   <zorder>textEdit_3</zorder>
   <zorder>textEdit_4</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1160</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>QVTKWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
